mappings/net/minecraft/server/MinecraftServer.mapping

```diff
@@ -1,9 +1,11 @@
 CLASS net/minecraft/server/MinecraftServer
 	FIELD field_2837 logger Ljava/util/logging/Logger;
-	FIELD field_2839 pendingConnectionManager Lnet/minecraft/class_9;
+	FIELD field_2839 packetHandlerManager Lnet/minecraft/class_9;
```

Comment:
what is a packet handler manager and why does that names sound cursed

Reply:
ceck #62